docs/register-intune-device-id.md

```diff
@@ -10,12 +10,12 @@ Users with a non-SE GSIB device and onboarding their Internet Device to SEED.
 
 ## Prerequisites
 
```

Comment:
the capitalisation for the Title to align with sidebar

Reply:
done